src/infrastructure/services/nft/NftService.ts

```diff
@@ -48,6 +49,24 @@ const nftApi = api.injectEndpoints({
         },
       }),
     }),
+    buyNftByPack: builder.mutation({
+      query: () => ({
+        url: `${process.env.REACT_APP_FTX_API_URL}/nft/packs/3/buy`,
```

Comment:
Would be a good idea to send packId as parameter here or use a constant in a separate file just to avoid touching service layer in future

Reply:
Yeah, I was thinking the same thing!